### PR TITLE
crypto: Correctly convert passphrases from Python to C

### PR DESCRIPTION
**Pull request by vojtechtrefny**
We can't use ord to convert the Python string to the byte array expected, with ord we'd support only ASCII passphrases and we want to support all unicode characters.

### CODE REVIEW

**Review by tbzatek — APPROVED** [review]
Yeah, makes sense to default to UTF-8. Gtk2+ is UTF-8 only and hope that everything modern Web-like too.

I know these are Python bindings but would be perhaps nice to document this, if there's a chance.

File: tests/crypto_test.py [hunk 1]
```diff
@@ -448,6 +448,43 @@ def test_luks_open_close(self):
     def test_luks2_open_close(self):
         self._luks_open_close(self._luks2_format)
 
+    @tag_test(TestTags.SLOW, TestTags.CORE)
+    def test_luks2_open_close_non_ascii_passphrase(self):
+        passphrase = "šššššššš"
```
Comment (vpodzime) [inline tests/crypto_test.py:453]:
```suggestion
        passphrase = "šššššššš-hů-hů"
```

**Review by vpodzime — APPROVED** [review]
ACK (with my critical fix, of course!)

File: tests/crypto_test.py (hunk 1 above)
Comment (tbzatek) [reply tests/crypto_test.py:453]: Good point! Go for it!

**Pull request merged**